src/doc/gui_manual/Operators/OperatorTypes/PICS/IntegralCurve_operator.rst

```diff
@@ -0,0 +1,298 @@
+    * Normal - X Y Z vector. The "normal" of the plane.
+    * Up Axis - X Y Z vector. The "up axis" serves as the "Y" axis embedded in the plane. (The vector orthogonal to both the "up axis" and the "normal" serves as the "X" axis embedded in the plane.)
+    * Sampling - Specify the type of sampling, the region from which to sample (interior or boundary) and the number of samples.
+       * The sampling "Distance in X" and "Distance in Y" define the dimensions of the rectangle in the plane. 
+   
+Sphere
+    Seed from a sphere defined by a point and a radius. The sphere location and size can be set interactively using the interactive sphere tool.
+
+    * Origin - X Y Z location of the origin.
+    * Radius - radius of the sphere.
+    * Sampling - Specify the type of sampling, the region from which to sample (interior or boundary) and the number of samples.
+   
+       * "Samples in Latitude", "Samples in Longitude", and "Samples in R" specifies how many samples to take along each direction.
+   
+Box
+    Seed from a box defined by bounds along the X, Y, and Z axes. The box location and size can be set interactively using the interactive box tool.
+
+    * The "Whole data set" check box indicates that the seeding box should be the same as the bounding box of the entire dataset.
+    * X Extents - extents along the X axis.
+    * Y Extents - extents along the Y axis.
+    * Z Extents - extents along the Z axis. 
+    * Sampling - Specify the type of sampling, the region from which to sample (interior or boundary) and the number of samples.
+   
+Selection
+    Seed with a named selection.
+    
+Field Data
+    The seed points are defined by another operator and passed to the Integral Curve operator. The array containing the seed point(s) must begin its name with "Seed Points".
+
+    * Copy to point list - Copy the points to the point list source (see above). This allows points to be edited. 
+
+
+Sampling type
+"""""""""""""
+
+For samples taken from a line, circle, plane, sphere or box, there is an option to generate uniform or random samples from the specified region. In more detail:
+
+Uniform
+    Sample uniformly along the boundary or the interior. 
+
+    * Samples in X - (Default 1) Create N samples along the X axis.
+    * Samples in Y - (Default 1) Create N samples along the Y axis.
+    * Samples in Z - (Default 1) Create N samples along the Z axis. 
+   
+Random
+    Sample randomly along the boundary or the interior. 
+
+    * Random number of samples (Default 1).
+    * Random number Seed - (Default 0) ensures that the same set of random samples is taken each time the plot is regenerated. 
+
+Boundary vs Interior Samples
+""""""""""""""""""""""""""""
+
+Samples from a circle, plane, sphere, or box can be either taken from the boundary, or the interior. For example, in the case of a plane sampling source, the samples can either lie along the edges of the planar region, or within the bounded rectangle, as shown below.
```

Comment:
Similarly here, remove explicit ref to the actual options in the GUI and use a more general term. If you find it necessary, you could *introduce* some term of your choosing at the top like, *seed object generators*, explaining just once there that that means things like `circle`, `plane`, `sphere`, etc. and then just use that term everywhere you would ordinarily use the list.